### PR TITLE
feat : added the map component with all its properties

### PR DESCRIPTION
**Pull request by hibabdelk**

### CODE REVIEW

**Comment by osamaakb** [comment]
please resolve all the conflicts then push again

File: src/App.js [hunk 1]
```diff
@@ -1,6 +1,4 @@
 import 'antd/dist/antd.css';
-import ButtonComponont from './Components/./Button/Button';
-
 function App() {
```
Comment (osamaakb) [inline src/App.js:2]: I can't merge the app.js like this, please remove it from the PR and make sure you have the latest master version

File: src/Components/Map/Map.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        >
+          <PopupInfo info={popupInfo} infoType={infoType} />
+        </Popup>
+      )}
+
+      <GeolocateControl style={geolocateStyle} />
+      <FullscreenControl style={fullscreenControlStyle} />
+      <NavigationControl style={navStyle} />
+      <ScaleControl style={scaleControlStyle} />
+      <ControlPanel onSelectCity={onSelectCity} />
```
Comment (osamaakb) [inline src/Components/Map/Map.js:93]: is this map for the cities page only? or will it be used somewhere else?

File: src/Components/Map/Markers/Markers.css [hunk 3]
```diff
@@ -0,0 +1,58 @@
+.tooltiptext {
+  visibility: visible;
+  width: 100px;
+  color: #fff;
+  text-align: center;
+  border-radius: 6px;
+  position: absolute;
+  z-index: 0;
+  bottom: 125%;
```
Comment (osamaakb) [inline src/Components/Map/Markers/Markers.css:9]: why is this very big value?

**Review by osamaakb — CHANGES_REQUESTED** [review]

File: src/Components/Map/Map.js (hunk 2 above)
Comment (hibabdelk) [reply src/Components/Map/Map.js:93]: It is for the activities and city pages, the width, height zoom and all the main properties of the map are set as props as well as the Markers' style in the Marker folder

File: src/Components/Map/Markers/Markers.css (hunk 3 above)
Comment (hibabdelk) [reply src/Components/Map/Markers/Markers.css:9]: I'm not sure why actually I took the Css from w3schools and when I sat it to a lower value the popup info shows on the top of the tooltip and it doesn't work as it supposed to

File: src/App.js (hunk 1 above)
Comment (hiba28) [reply src/App.js:2]: Yup done

File: src/Components/Map/Map.js (hunk 2 above)
Comment (osamaakb) [reply src/Components/Map/Map.js:93]: ok then why are you passing `onSelectCity` in the props?

File: src/Components/Map/Markers/Markers.css (hunk 3 above)
Comment (osamaakb) [reply src/Components/Map/Markers/Markers.css:9]: Ok

**Comment by osamaakb** [comment]
please also resolve these conflicts here

File: src/Components/Map/Map.js (hunk 2 above)
Comment (hibabdelk) [reply src/Components/Map/Map.js:93]: I changed the functionality of it many times and I forgot to change the name I'll do it right away

**Pull request closed without merging**